### PR TITLE
DEVX-1973 DEVX-1974 DEVX-1976 Docs conversion: ruby ksql-datagen nodejs

### PR DESCRIPTION
**Pull request by tikimims**

### CODE REVIEW

File: clients/cloud/ksql-datagen/README.md [hunk 1]
```diff
@@ -1,149 +1,9 @@
 # Overview
 
-Produce messages to and consume messages from a Kafka cluster using the [KSQL datagen](https://docs.confluent.io/current/ksql/docs/tutorials/generate-custom-test-data.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) command-line tool.
+Produce messages to and consume messages from a Kafka cluster using the [KSQL Datagen](https://docs.confluent.io/current/ksql/docs/tutorials/generate-custom-test-data.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) command-line tool.
 
 *Note: `ksql-datagen` is meant for development purposes only and is not suitable for a production environment*
```
Comment (ybyzek) [inline clients/cloud/ksql-datagen/README.md]: Let's remove this line

File: clients/docs/ksql-datagen.rst [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+KSQL Datagen
+============
+
+In this tutorial, you will run a KSQL Datagen client application that
+produces messages to and consumes messages from an |ak-tm| cluster.
+
+.. note::
+
+    You should use KSQL Datagen for development purposes only. It isn't suitable
```
Comment (ybyzek) [inline clients/docs/ksql-datagen.rst]:
```suggestion
    Use KSQL Datagen for development purposes only. It isn't suitable
```

File: clients/docs/ksql-datagen.rst [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+       curl -u <SR API KEY>:<SR API SECRET> https://<SR ENDPOINT>/subjects
+
+#. Verify the subject ``test2-value`` exists.
+
+   .. code-block:: text
+
+      ["test2-value"]
+
+#. View the schema information for subject `test2-value`. In the following
```
Comment (ybyzek) [inline clients/docs/ksql-datagen.rst]:
```suggestion
#. View the schema information for subject ``test2-value``. In the following
```

**Review by ybyzek — CHANGES_REQUESTED** [review]
Thanks.  Please rebase to latest 5.5.1-post so a docs build can pass.

File: clients/cloud/ksql-datagen/README.md (hunk 1 above)
Comment (tikimims) [reply clients/cloud/ksql-datagen/README.md]: Removed

Comment (tikimims) [reply clients/cloud/ksql-datagen/README.md]: Updated

File: clients/docs/ksql-datagen.rst (hunk 2 above)
Comment (tikimims) [reply clients/docs/ksql-datagen.rst]: Updated

File: clients/docs/ksql-datagen.rst (hunk 3 above)
Comment (tikimims) [reply clients/docs/ksql-datagen.rst]: Updated

**Comment by tikimims** [comment]
Closing this PR. The NEW PR is https://github.com/confluentinc/examples/pull/760

**Pull request closed without merging**